test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -120,7 +120,6 @@
     ("aten::segment_reduce", datetime.date(2022, 6, 30)),
     ("aten::_segment_reduce_backward", datetime.date(2022, 6, 30)),
     ("aten::empty.SymInt", datetime.date(9999, 1, 1)),
-    ("c10d::broadcast", datetime.date(2022, 6, 25)),
```

Comment:
Is there a reason to remove this one?

Reply:
It's already expired back in June. Let me extract this out to a separate PR.